web/ui/react-app/src/index.tsx

```diff
@@ -10,8 +10,10 @@ import { isPresent } from './utils';
 
 // Declared/defined in public/index.html, value replaced by Prometheus when serving bundle.
```

Comment:
I think you'll also need to touch https://github.com/prometheus/prometheus/blob/b40e254f25906edbe365b59daa5f1ee26569b4dc/web/ui/react-app/public/index.html#L19-L22 and https://github.com/prometheus/prometheus/blob/5538c64e8edaf770208e79214841a52d9d95e5b0/web/web.go#L410-L412